homeassistant/components/zha/number.py

```diff
@@ -577,7 +577,7 @@ class TimerDurationMinutes(ZHANumberConfigurationEntity, id_suffix="timer_durati
 
 @CONFIG_DIAGNOSTIC_MATCH(channel_names="ikea_airpurifier")
 class FilterLifeTime(ZHANumberConfigurationEntity, id_suffix="filter_life_time"):
-    """Representation of a ZHA timer duration configuration entity."""
+    """Representation of a ZHA filter lifetime configuration entity."""
```

Comment:
I'm not convinced that "lifetime" is clearer than duration.

Reply:
The original "Timer duration" was a copy-paste error.
The sensor is actually describing the remaining lifetime of a filter.